__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+const db = require("../db/connection");
+//const connection = require("../db/connection");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  if (db.end) db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("200: should return an array of all the topics objects with slug and description properties", () => {
+    const expectedTopics = [
+      {
+        description: "The man, the Mitch, the legend",
+        slug: "mitch",
+      },
+      {
+        description: "Not dogs",
+        slug: "cats",
+      },
+      {
+        description: "what books are made of",
+        slug: "paper",
+      },
+    ];
+
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toMatchObject(expectedTopics);
```

Comment:
Creating an expected topics array here definitely works and is fine for topics.

It might get a little painstaking using this approach as you move towards the articles tickets and it also means if the test data changes in any way that this test would fail.

Rarther than being explicit here with the expected response you could look around the topics array and use the .toHaveProperty jest matcher to check that the key exists on each of the objects.

for example you could use a test like this to check for the key and assert the data type the value should be 'expect(topic).toHaveProperty("description", expect.any(String)'